ui/page/root/trade_page.go

```diff
@@ -52,8 +52,13 @@ func NewTradePage(l *load.Load) *TradePage {
 			},
 		},
 	}
+	filteredTabTitles := tabTitles
+	if l.IsMobileView() {
+		// Remove the first item for mobile view
+		filteredTabTitles = filteredTabTitles[1:]
+	}
```

Comment:
Please add that we are removing the DEX tab cuz it's not supported on mobile yet.